bb_wrapper/models/pagamentos.py

```diff
@@ -74,23 +75,25 @@ def _set_data(cls, values):
         from ..services.pix import PixService
 
         key = values.get("chave")
-
         key_type = PixService().identify_key_type(key)
-        values["formaIdentificacao"] = key_type
+        values["formaIdentificacao"] = key_type.value
 
         if key_type == TipoChavePIX.telefone:
-            values["dddTelefone"] = int(key[:2])
-            values["telefone"] = int(key[2:])
+            values["dddTelefone"] = key[:2]
+            values["telefone"] = key[2:]
         elif key_type == TipoChavePIX.email:
-            values["email"] = key
+            if PixService().verify_email(key):
+                values["email"] = key
```

Comment:
esse verify_email está lançando uma exceção? Se sim, não precisa de if!

```suggestion
            PixService().verify_email(key)
            values["email"] = key
```